.eslintrc.js

```diff
@@ -400,10 +400,185 @@ module.exports = {
+        // These will be turned on after bug fixes are mostly completed and we can
+        // run a fix-lint
+        /*
+        'import/order': [
+          'warn',
+          {
+            groups: ['builtin', 'external', 'internal', 'parent', 'sibling', 'index'],
+            'newlines-between': 'always',
+          },
+        ],
+        */
+        'no-bitwise': 'error',
+        'no-continue': 'error',
+        'no-dupe-keys': 'error',
+        'no-duplicate-case': 'error',
+        // This will be turned on after bug fixes are mostly completed
+        // 'no-duplicate-imports': 'warn',
+        'no-empty-character-class': 'error',
+        'no-empty-pattern': 'error',
+        'no-ex-assign': 'error',
+        'no-extend-native': 'error',
+        'no-extra-bind': 'error',
+        'no-extra-boolean-cast': 'error',
+        'no-extra-label': 'error',
+        'no-floating-decimal': 'error',
+        'no-func-assign': 'error',
+        'no-implicit-globals': 'error',
+        'no-implied-eval': 'error',
+        'no-invalid-regexp': 'error',
+        'no-inner-declarations': 'error',
+        'no-lone-blocks': 'error',
+        'no-multi-assign': 'error',
+        'no-misleading-character-class': 'error',
+        'no-new-symbol': 'error',
+        'no-obj-calls': 'error',
+        // This will be turned on after bug fixes are mostly complete
+        // 'no-param-reassign': 'warn',
+        'no-process-exit': 'error',
+        'no-prototype-builtins': 'error',
+        // This will be turned on after bug fixes are mostly complete
+        // 'no-return-await': 'warn',
+        'no-self-compare': 'error',
+        'no-shadow-restricted-names': 'error',
+        'no-sparse-arrays': 'error',
+        'no-this-before-super': 'error',
+        // This will be turned on after bug fixes are mostly complete
+        // 'no-undef': 'warn',
+        'no-unreachable': 'error',
+        'no-unsafe-finally': 'error',
+        'no-useless-call': 'error',
+        // This will be turned on after bug fixes are mostly complete
+        // 'no-useless-catch': 'warn',
+        'no-useless-concat': 'error',
+        'no-useless-computed-key': 'error',
+        // This will be turned on after bug fixes are mostly complete
+        // 'no-useless-escape': 'warn',
+        'no-useless-rename': 'error',
+        // This will be turned on after bug fixes are mostly complete
+        // 'no-useless-return': 'warn',
+        // This will be turned on after bug fixers are mostly complete
+        // 'no-void': 'warn',
+        'one-var-declaration-per-line': 'error',
+        'prefer-object-spread': 'error',
+        'prefer-promise-reject-errors': 'error',
+        'prefer-rest-params': 'error',
+        'prefer-spread': 'error',
+        // This style will be turned on after most bugs are fixed
+        // 'prefer-template': 'warn',
+        // This style will be turned on after most bugs are fixed
+        // quotes: ['warn', 'single', { avoidEscape: true }],
```

Comment:
There've been plenty of PR comments for this one. ++ for this rule.

Reply:
Soon for ones such as that one.